src/PresentationalComponents/RenderForms.js

```diff
@@ -48,36 +48,43 @@ const FormTemplateWrapper = (props) => (
   <FormTemplate {...props} submitLabel="Save" canReset />
 );
 
-const RenderForms = ({ schemas, loaded, saveValues, ...props }) => (
-  <Stack {...props} hasGutter>
-    {loaded ? (
-      schemas.map((schema, i) => (
-        <StackItem key={`settings-form-${i}`}>
+const RenderForms = ({ schemas, loaded, saveValues, ...props }) => {
+  const [initialValues, setInitialValues] = useState();
+  return (
+    <Stack {...props} hasGutter>
+      {loaded ? (
+        schemas.map((schema, i) => (
+          <StackItem key={`settings-form-${i}`}>
+            <Card>
+              <CardBody>
+                <FormRender
+                  componentMapper={componentMapperExtended}
+                  FormTemplate={FormTemplateWrapper}
+                  schema={schema}
+                  submitLabel="Save"
+                  onSubmit={(values) => {
+                    setInitialValues(values);
+                    return saveValues(values);
```

Comment:
In order to `re-initialize` the form, you also have to initialize the internal state. You can do that by calling this:
```jsx
onSubmit={(values, formApi) => {
                    setInitialValues(values);
                    formApi.initialize(values);
                    return saveValues(values);
            }}
```

Otherwise, you may run into issues with some of the form state flags being invalid.